pkg/userdata/coreos/provider_test.go

```diff
@@ -264,6 +264,29 @@ func TestUserDataGeneration(t *testing.T) {
 				DisableAutoUpdate: true,
 			},
 		},
+		{
+			name: "v1.15.0-vsphere",
+			providerSpec: &providerconfig.Config{
+				CloudProvider: "vsphere",
```

Comment:
Why aren't SSH public keys needed here? :thinking:

Reply:
Well, they are not a mandatory property and I thought the fixture could be without a SSH key :)